docs/getting-started-step-by-step/00-overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview Getting Started: Step-by-Step
+
+The following guide provides an easy, step-by-step path for developers and platform engineers to get familiar with Crossplane. It complements the current Getting Started guide, which focuses on getting an RDS Instance up and running quickly. In this example will use Object Storage as a simple example with quick turn around times.
+
+In this guide we are going to all important aspects of Crossplane: Managed Resources, Provider, ProviderConfigs, Compositions, CompositeResourceDefinitions (XRD), CompositeResources (XR), Claims and Configurations. 
+
+Here is the full architectural overview: 
+
+![architecture-overview](00-architecture-overview.jpg)
+
+Don't worry about the different terms. We will introduce them on our way.
+
+We will use this condensed form throughout the guide: 
+
+![architecture-overview-condensed](architecture-all.jpg)
+
+## Parts & Steps
+
+The guide consists of 4 parts with individual steps:
+
+* 1.) [Setup](01-setup.md): Setting up Crossplane on Kubernetes.
+    * Access a Kubernetes cluster
+    * Install the Crossplane CLI
+    * Install the Crossplane Runtime
+* 2a.) [Using Managed Resources from AWS](02a-managed-resources-aws.md): Connect Crossplane to manage AWS resources.
+    * Install the AWS Provider 
+    * Configure the AWS Provider
+    * Create, Update and Delete an S3 bucket
+* 2b.) Using Managed Resources from GCP: Connect Crossplane to manage GCP resources.
+    * Install the GCP Provider 
+    * Configure the GCP Provider
+    * Create, Update and Delete a Cloud Storage bucket
+* 3.) Building a Composite Resource: Build and use simplified and catered infrastructure APIs.
+    * Create a CompositeResourceDefinition (XRD)
+    * Create a CompositeResource (XR) which implements the XRD
```

Comment:
Could we flip this and say that the XR is defined by the XRD? We try to mirror the Kubernetes CR to CRD relationship here and I don't think it's common to say that a CR "implements" its definition.

Reply:
I'm happy to follow the official wording. And as it explains a core concept it needs to be consistent.

That said when I read "XR is defined by the XRD" I interpret that the XRD knows about the XR. The association implements comes from developer experience.

Reply:
It's arguably more accurate to say that the XRD knows about the XR than that the XR knows about the XRD. The relationships is like a custom resource and a custom resource definition or a programming language type definition and an instance of a type. The latter is defined into existence by the former.